From: Jan Engelhardt <jengelh@inai.de>
Date: 2017-06-07 13:48:49.472854532 +0200

[   53s] htapp.cc:3026:18: error: call of overloaded 'abs(uint)' is ambiguous
[   53s] /usr/include/c++/7/bits/std_abs.h:56:3: note: candidate: long int std::abs(long int)
[   53s] /usr/include/c++/7/bits/std_abs.h:61:3: note: candidate: long long int std::abs(long long int)
[   53s] /usr/include/c++/7/bits/std_abs.h:70:3: note: candidate: constexpr double std::abs(double)
[   53s] /usr/include/c++/7/bits/std_abs.h:74:3: note: candidate: constexpr float std::abs(float)
[   53s] /usr/include/c++/7/bits/std_abs.h:78:3: note: candidate: constexpr long double std::abs(long double)
[...]

---
 htapp.cc |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

Index: ht-2.1.0/htapp.cc
===================================================================
--- ht-2.1.0.orig/htapp.cc
+++ ht-2.1.0/htapp.cc
@@ -3023,7 +3023,7 @@ static uint isqr(uint u)
 {
 	uint a = 2;
 	uint b = u/a;
-	while (abs(a - b) > 1) {
+	while (abs(static_cast<long>(a - b)) > 1) {
 		a = (a+b)/2;
 		b = u/a;
         }
